cellpack/autopack/__init__.py

```diff
@@ -411,14 +411,12 @@ def load_file(
         collection = file_path.split("/")[0]
         db_doc, _ = db_handler.collect_docs_by_id(collection=collection, id=recipe_id)
         downloaded_recipe_data = db_handler.prep_db_doc_for_download(db_doc)
-        if collection == "recipes_edited":
-            database_name = "firebase_edited"
-        return downloaded_recipe_data, database_name
+        return downloaded_recipe_data, database_name, collection == "recipes_edited"

```

Comment:
returning `collection == "recipes_edited"` here might not be immediately clear to first time reviewers regarding how this relates to "unnested". how about using something like `is_unnested_firebase` or `unnested_collection` to clarify our intent?

Reply:
- [ ] add local variable to clarify what's happening